### DISCUSSION TITLE
Navigation element bug

### DISCUSSION THREAD

**Discussion by crizmo** [community/community#73111] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Whenever i navigate to like someone's profile / my profile or to any repo or anywhere on GitHub
this thing pops up for a second or two on the right corner

![image](https://github.com/community/community/assets/83665497/6a25b02a-9135-4d52-84c7-f0bac8b5b801)

Then it goes. 
I don't know if its a bug or what

**Comment by crizmo** [comment]
The issue or bug element is still present, and no help has been provided so far.

**Comment by steveqobs** [comment]
me too
